DISCLAIMER.md

```diff
@@ -0,0 +1,51 @@
+# Disclaimer
+
+Simple Icons asks that its users read this disclaimer fully before including an icon in their project.
+
+## Table of Contents
+
+* [Licenses, Copyrights & Trademarks](#licenses-copyrights--trademarks)
+* [Brand Guidelines](#brand-guidelines)
+* [Update of Brands](#update-of-brands)
+* [Removal of Brands](#removal-of-brands)
+
+## Licenses, Copyrights & Trademarks
+
+> NOTE:
+> - The addition of licenses to Simple Icons is an ongoing project. Hence, the absence of licence data for a particular icon does not imply that the icon is not released under a license.
+> - Simple Icons is released under CC0 - though that doesn't mean to imply that all icons within the project are also CC0. Please see individual licenses where available.
+
+Simple Icons provides data on the license under which icons are available. We ask users to carefully consider this when using an icon.  As licenses are subject to change we also ask our users to regularly check if the license of the icons they use have been changed.
+
+The license under which an icon is available can be found in the icon's entry in the JSON data file in the NPM and Packagist packages.
+
+As we rely largely on our community to help us keep Simple Icons up-to-date, the license data for an icon may be outdated. If you find that the license data provided is outdated, please [submit an issue] to [the Simple Icons GitHub repository].
+
+If an icon includes a registered trademark (`®`) or trademark symbol (`™`) the recommendations outlined in [the Simple Icons Contributing Guidelines] are followed to decide whether to include the symbol or not.
+
+Simple Icons cannot be held responsible for any legal activity raised by a brand, or users of the package. We ask that our users seek the correct permissions to use the icons relevant to their project.
+
+## Brand Guidelines
+
+> NOTE: the addition of guidelines to Simple Icons is an ongoing project. In the meantime, users of Simple Icons are instead encouraged to check the `source` URL as, in some cases, the icon will have been sourced from official guidelines. The lack of a `guidelines` entry for a particular brand does not imply that the brand has no guidelines.
+
+Simple icons provides a link to a brand's _branding guidelines_ (or similar) if the brand provides one. We ask our users read these guidelines and ensure their usage of the brand's icon is in accordance with them. As guidelines are subject to change we also ask our users to regularly check if the brand guidelines of the icons they use have been updated.
```

Comment:
Minor but we are writing our name as "Simple Icons" everywhere so for consistency:

```suggestion
Simple Icons provides a link to a brand's _branding guidelines_ (or similar) if the brand provides one. We ask our users read these guidelines and ensure their usage of the brand's icon is in accordance with them. As guidelines are subject to change we also ask our users to regularly check if the brand guidelines of the icons they use have been updated.
```